content/open-source.fr.md

```diff
@@ -2,12 +2,95 @@
 	"title": "Open Source",
 	"type": "page",
 	"aliases": ["opensource", "oss", "foss"],
-	"heading": "<span style='text-align: center; font-family: monospace;'>demandetesdonnees.fr <span class='color-red-600' title='liebt'>❤</span> Open Source</span>"
+	"heading": "<span style='text-align: center; font-family: monospace;'>demandetesdonnees.fr <span class='color-red-600' title='aime'>❤</span> l'Open Source</span>"
 }
 
-<!-- TODO: Translate. -->
+<img id="open-source-humaaan" class="top-left-humaaan" style="margin-top: -70px;" src="/img/humaaans/open-source.svg">
+
+L'Open Source est au cœur même de demandetesdonnees.fr. Nous avons conçu le projet à partir de rien pour qu'il soit le plus ouvert possible. Notre [constitution](https://www.datarequests.org/verein/constitution/) nous engage à publier notre contenu sous licence libre.
```

Comment:
```suggestion
L'Open Source est au cœur même de demandetesdonnees.fr. Nous avons conçu le projet à partir de rien pour qu'il soit le plus ouvert possible. Notre [constitution]({{< ref "verein/constitution" >}}) nous engage à publier notre contenu sous licence libre.
```

Absolutely, that is a relic from before you had translated that to avoid an undefined reference.